services/tasks.py

```diff
@@ -95,7 +126,18 @@ def run(self, service_id, **kwargs):
             status = self.get_health(service.url, service.token)
             try:
                 result = status.json()
+
+                if (not service.up and result['up'] and service.last_up_at):
+                    downtime = timezone.now() - service.last_up_at
+                    fire_metric.apply_async(kwargs={
+                        "metric_name": 'services.downtime.%s.sum' % (
+                            utils.normalise_string(service.name)),
+                        "metric_value": downtime.seconds // 60
```

Comment:
This won't give us our desired results. We need to `sum` the value since the last check, not since last up time.

Otherwise, let's say we check every second, and the service is down for 5 seconds. Then we would be firing the values, `1`, `2`, `3`, `4`, `5`. When that value gets aggregated, using the `sum` aggregator, we will get a value of `1 + 2 + 3 + 4 + 5 = 15`, but the service wasn't down for 15 seconds, it was down for 5.

For this kind of metric, we want the `last` aggregator, which just takes the last value for the bucket. For our `sum` aggregator, we just want to fire the time since the last check.